.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,13 @@
+<!-- Please update this template with something that matches your PR -->
+| Q             | A
+| ------------- | ---
+| Bug fix?      | yes/no
+| New feature?  | yes/no
+| BC breaks?    | no
+| Deprecations? | yes/no
+| Tests pass?   | yes
+| Fixed tickets | close #...
```

Comment:
We should follow the same template as Symfony IMO: https://raw.githubusercontent.com/symfony/symfony/master/.github/PULL_REQUEST_TEMPLATE.md

Reply:
Done